### DISCUSSION TITLE
how to integrate it into Laravel ?

### DISCUSSION THREAD

**Discussion by FloRobart** [prasathmani/tinyfilemanager#1201] [Q&A]
I am creating my own cloud with Laravel and I would like to know how to integrate your file manager into my Laravel project because I am having some difficulty.

**Comment by prasathmani** [comment]
https://github.com/prasathmani/tinyfilemanager/wiki/Embedding

**Reply by FloRobart** [reply]
thank you but I'm a beginner and I have this message that appears and I don't understand why.

```php
Tiny File Manager
Error: Cannot load configuration
```
